pkg/coscheduling/coscheduling.go

```diff
@@ -137,6 +138,30 @@ func (cs *Coscheduling) PreFilter(ctx context.Context, state *framework.CycleSta
 	return framework.NewStatus(framework.Success, "")
 }
 
+// PostFilter is use for rejecting the group pods if a pod can not pass resource check when filter.
+func (cs *Coscheduling) PostFilter(ctx context.Context, state *framework.CycleState, pod *v1.Pod,
+	filteredNodeStatusMap framework.NodeToStatusMap) (*framework.PostFilterResult, *framework.Status) {
+	// check if failed due to preFilter failed, if not, filteredNodeStatusMap should not be nil
```

Comment:
This comment doesn't look correct - PreFilter would return `Unschedulable` in most cases, then filteredNodeStatusMap is populated on each node:

https://github.com/kubernetes/kubernetes/blob/v1.19.5/pkg/scheduler/core/generic_scheduler.go#L273-L275